doc/sr_plugins.dox

```diff
@@ -4,7 +4,9 @@
 Sysrepo itself supports 2 kinds of plugins not to be confused with `sysrepo-plugind` plugins. **Datastore plugins**
 define a set of callbacks that implement all the operations performed on a datastore allowing to implement a completely
 custom datastore. Every datastore for every YANG module can use a different datastore implementation. **Notification
-plugins** is the exact same concept used for implementing storage of notifications stored for replay.
+plugins** is the exact same concept used for implementing storage of notifications stored for replay. These plugins can
```

Comment:
In general, `sysrepoctl` and `sysrepocfg` tools are not directly mentioned in the docs because they are just useful utilities that call *sysrepo* API functions and have their own `help`.